src/controllers/location.js

```
@@ -0,0 +1,18 @@
+const { getCountries, getCitiesByCountry } = require('../services/location')
+
+const getCountriesList = async (req, res) => {
+  const { search = '' } = req.query
+  const countries = await getCountries(search)
+  res.json(countries)
+}
+
+const getCitiesList = async (req, res) => {
+  const { country } = req.params
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)